README.md

```diff
@@ -14,49 +14,54 @@ cargo run --release -- help
 <details><summary>example output</summary>
 
 ```commandline
-Casper client 1.5.0
+Casper client 2.0.0
 A client for interacting with the Casper network
 
 USAGE:
     casper-client [SUBCOMMAND]
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+OPTIONS:
+    -h, --help       Print help information
+    -V, --version    Print version information
```

Comment:
This change makes the strings not consistent with the ones attached to SUBCOMMANDS.

In general I'm very easy on such changes, but in the other place I see that we change the strings the opposite way: https://github.com/casper-ecosystem/casper-client-rs/pull/16/files#diff-4d07a1db2fe90e5991b5221aabc68348d09e85c7a86ff816aa32e83b55795f64R27

So just bringing this to your attention.

Reply:
Fixed in [c5cdb6a](https://github.com/casper-ecosystem/casper-client-rs/pull/16/commits/c5cdb6afe51143fcf058ac6184e67d21233cd776).